src/index.ts

```diff
@@ -66,6 +66,10 @@ export interface IClientConfigSync {
   enableOverrides?: boolean;
 
   overridesStorageKey?: string;
+
+  configPublishedAt?: string; // ISO Datetime String
```

Comment:
The DX of having to drill into the UFC for `offlineInit` is poor, but this is the only way currently and there's already precedent for passing around the flag entities directly instead of a wrapped config object. `ConfigWire` will solve this, so this is an ok workaround for now.

Reply:
I was just thinking how `ConfigWire` should make this all way better